packages/utils/src/lib/files/excel.ts

```diff
@@ -0,0 +1,97 @@
+import type {
+  ColInfo,
+  JSON2SheetOpts,
+  WorkBook,
+  WorkSheet,
+  WritingOptions
+} from 'xlsx';
+
+/**
+ *
+ * @param ws workSheet SheetJs definition https://docs.sheetjs.com/docs/csf/sheet/
+ * @param wsname Records reprensenting the dataset
+ * @returns workBook SheetJs definition of an excel file
+ */
+export async function createExcelWorkBook(
+  workSheet?: WorkSheet,
+  wsname?: string
+): Promise<WorkBook> {
+  const { utils } = await import('xlsx');
+
+  return utils.book_new(workSheet, wsname);
+}
+/**
+ *
+ * @param title The sheet title
+ * @param rows Records reprensenting the dataset
+ * @param workBook workBook SheetJs definition of an excel file
+ * @param opts options to convert json to sheet, refer to SheetJs https://docs.sheetjs.com/docs/api/utilities/array#array-of-objects-input
+ */
+export async function addExcelSheetToWorkBook<T = Record<string, unknown>>(
+  title: string,
+  rows: T[],
+  workBook: WorkBook,
+  jsonToSheetOpts?: JSON2SheetOpts,
+  bookAppendSheetOpts: { roll?: boolean } = { roll: undefined }
```

Comment:
La règle du pouce, on ne devrait pas dépasser 3 paramètres par fonction. Est-ce qu'on pourrait au moins rassembler les deux options en une seule `jsonToSheetOpts` et `bookAppendSheetOpts`

Reply:
ok.

Reply:
noté.